### DISCUSSION TITLE
大佬能否出个x-ui加caddy+naiveproxy谢谢了

### DISCUSSION THREAD

**Discussion by MINGERTAI** [lxhao61/integrated-examples#104] [Q&A] [answered]
希望能出个x-ui加caddy+naiveproxy谢谢了
    这样就能方便很多小白配置x-ui加caddy+naiveproxy，x-ui各种回落，路径的设置问题了

                                                                       麻烦大佬了，谢谢🙏

**Comment by MINGERTAI** [comment]
也就是说同时安装x-ui 与您的编译好caddy（加各种插件的caddy），x-ui回落，路径都由您编译的caddy来实现，加naiveproxy与trojan go 功能的caddy.json配置

**Comment by lxhao61** [comment, marked as answer]
根本无必要！x-ui是Xray套壳，你要实现那些应用直接对应替换即可。
只是注意：
1、无法使用本地监听、UDS监听及回落本地、回落UDS。
2、无法自身实现单端口多用户。
3、无法灵活配置。例如实现  [V2Ray[SS+(DD+WebSocket)]+Nginx\Caddy](https://github.com/lxhao61/integrated-examples/tree/main/V2Ray%5BSS%2B(DD%2BWebSocket)%5D%2BNginx%5CCaddy)  等。
